### DISCUSSION TITLE
Semantic segmentation support

### DISCUSSION THREAD

**Discussion by patrontheo** [obss/sahi#1063] [Ideas]
Hello all,
Your framework is really useful when working with object detection models, as I work with very large aerial images.
I was wondering if you plan to support semantic segmentation in the future. I read here #788 that it wasn't supported but the post is from 2022 so I was wondering if you had any updates about that.
Thanks !

**Comment by fcakyon** [comment]
SAHI currently supports instance segmentation but not semantic segmentation.

**Reply by patrontheo** [reply]
Yeah that's what I thought. 
Do you plan to support semantic segmentation in the future ?
Thanks
